orsopy/fileio/base.py

```diff
@@ -33,8 +44,14 @@ def _clean(self):
         :return: Cleaned dictionary
         :rtype: dict
         """
-        return dict((k, v) for (k, v) in self.__dict__.items()
-                    if (v is not None or k not in self.orso_optionals))
+        d = {}
+        for k, v in self.__dict__.items():
+            if (v is not None or k not in self.orso_optionals):
```

Comment:
I think this could include callable objects if they are set during instantiation and are not in orso_optionals.